data/alpha/field/enacting-legislation.yaml

```diff
@@ -0,0 +1,6 @@
+text: The legislation that created a council area in Scotland.
```

Comment:
'created', or 'led to the creation of' -- seems inconsistent with `dissolving-legislation`.